elixir-graphql/san_graphql_ex/README.md

```diff
@@ -0,0 +1,21 @@
+# SanGraphqlEx
+
+**TODO: Add description**
+
+## Installation
+
+If [available in Hex](https://hex.pm/docs/publish), the package can be installed
+by adding `san_graphql_ex` to your list of dependencies in `mix.exs`:
+
+```elixir
+def deps do
+  [
+    {:san_graphql_ex, "~> 0.1.0"}
```

Comment:
we can add `github: ` until available on hex.pm ?

Reply:
oops, this should have been deleted.